it/controllers/IndexControllerISpec.scala

```diff
@@ -466,14 +796,80 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
               status = SubmissionStatusEnum.Submitted
             )
           )
-        )),
+          )),
         communications = Seq.empty,
         financial = None
       )
     ),
     otherPenalties = Some(true)
   )
 
+  val getPenaltyDetailsPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue: GetPenaltyDetails = getPenaltyDetailsPayloadWithAddedPoint.copy(
+    totalisations = Some(Totalisations(
+      LSPTotalValue = Some(400),
+      penalisedPrincipalTotal = Some(121.40),
+      LPPPostedTotal = Some(165.25),
+      LPPEstimatedTotal = Some(46.55),
+      LPIPostedTotal = Some(40.55),
+      LPIEstimatedTotal = Some(6)
+    )),
+
+    lateSubmissionPenalty = Some(LateSubmissionPenalty(
+      summary = LSPSummary(
+        activePenaltyPoints = 2,
+        regimeThreshold = 4,
+        inactivePenaltyPoints = 0,
+        penaltyChargeAmount = 0
+      ),
+      details = Seq(
+        LSPDetails(
+          penaltyNumber = "1234567890",
+          penaltyOrder = "01",
+          penaltyCategory = LSPPenaltyCategoryEnum.Point,
+          penaltyStatus = LSPPenaltyStatusEnum.Active,
+          FAPIndicator = Some("X"),
```

Comment:
Wouldn't this mean it was an added point? (if not intended, set to None)

Reply:
Done, set to None